test/table.ts

```diff
@@ -2682,7 +2682,7 @@ describe('Bigtable/Table', () => {
 
       it('should not retry unretriable errors', done => {
         const unretriableError = new Error('not retryable') as ServiceError;
-        unretriableError.code = 10; // Aborted
+        unretriableError.code = 3; // INVALID_ARGUMENT
```

Comment:
I see. Looks like this change had to be made because of the new retry code we are allowing.